packages/quick_actions/quick_actions/android/src/main/java/io/flutter/plugins/quickactions/QuickActionsPlugin.java

```diff
@@ -6,15 +6,19 @@
 
 import android.app.Activity;
 import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
 import io.flutter.embedding.engine.plugins.FlutterPlugin;
 import io.flutter.embedding.engine.plugins.activity.ActivityAware;
 import io.flutter.embedding.engine.plugins.activity.ActivityPluginBinding;
 import io.flutter.plugin.common.BinaryMessenger;
 import io.flutter.plugin.common.MethodChannel;
+import io.flutter.plugin.common.PluginRegistry.NewIntentListener;
 
 /** QuickActionsPlugin */
-public class QuickActionsPlugin implements FlutterPlugin, ActivityAware {
+public class QuickActionsPlugin implements FlutterPlugin, ActivityAware, NewIntentListener {
   private static final String CHANNEL_ID = "plugins.flutter.io/quick_actions";
+  protected static boolean isInitialized = false;
```

Comment:
Why is this static? Wouldn't each instance of the plugin need to be initialized?

Reply:
sounds like the use case is to support a single initialization.  If that is the case, it could still be an instance field. 

That said, I don't there's such a case as registering/initializing the same plugin name more than once.

Reply:
Removed as it's unnecessary. Thanks.